src/annotator/integrations/test/pdf-metadata-test.js

```diff
@@ -202,135 +202,114 @@ describe('PDFMetadata', function () {
-          ],
-          documentFingerprint: fakePDFViewerApplication.pdfDocument.fingerprint,
-        };
-
-        return pdfMetadata.getMetadata().then(function (actualMetadata) {
-          assert.deepEqual(actualMetadata, expectedMetadata);
-        });
+    it('returns the document fingerprint in the `links` array', async () => {
+      const { pdfMetadata } = createPDFMetadata();
+      const metadata = await pdfMetadata.getMetadata();
+      assert.deepInclude(metadata.link, {
+        href: `urn:x-pdf:${testMetadata.fingerprint}`,
       });
+    });
 
-      it('gets the title from the documentInfo.Title field', function () {
-        const expectedMetadata = {
-          title: fakePDFViewerApplication.documentInfo.Title,
-          link: [
-            {
-              href:
-                'urn:x-pdf:' + fakePDFViewerApplication.pdfDocument.fingerprint,
-            },
-            { href: fakePDFViewerApplication.url },
-          ],
-          documentFingerprint: fakePDFViewerApplication.pdfDocument.fingerprint,
-        };
-
-        fakePDFViewerApplication.metadata.has = sinon.stub().returns(false);
-
-        return pdfMetadata.getMetadata().then(function (actualMetadata) {
-          assert.deepEqual(actualMetadata, expectedMetadata);
-        });
+    it('does not return file:// URLs in `links` array', async () => {
+      const { pdfMetadata } = createPDFMetadata({
+        fingerprint: 'fakeFingerprint',
+        url: 'file://fake.pdf',
       });
 
-      it('does not save file:// URLs in document metadata', function () {
-        let pdfMetadata;
-        const fakePDFViewerApplication = new FakePDFViewerApplication();
-        fakePDFViewerApplication.completeInit();
-        fakePDFViewerApplication.finishLoading({
-          fingerprint: 'fakeFingerprint',
-          url: 'file://fake.pdf',
-        });
-        const expectedMetadata = {
-          link: [
-            {
-              href:
-                'urn:x-pdf:' + fakePDFViewerApplication.pdfDocument.fingerprint,
-            },
-          ],
-        };
-
-        pdfMetadata = new PDFMetadata(fakePDFViewerApplication);
-
-        return pdfMetadata.getMetadata().then(function (actualMetadata) {
-          assert.equal(actualMetadata.link.length, 1);
-          assert.equal(
-            actualMetadata.link[0].href,
-            expectedMetadata.link[0].href
-          );
-        });
+      const metadata = await pdfMetadata.getMetadata();
+
+      const fileLink = metadata.link.find(link =>
+        link.href.includes('file://')
+      );
+      assert.isUndefined(fileLink);
+    });
+
+    it('gets the title from the dc:title field', async () => {
+      const { pdfMetadata } = createPDFMetadata();
+      const metadata = await pdfMetadata.getMetadata();
+      assert.equal(metadata.title, testMetadata.metadata['dc:title']);
+    });
+
+    it('gets the title from the documentInfo.Title field', async () => {
```

Comment:
It might be helpful to detail what context causes the `title` to come from `documentInfo.title` vs. `dc:title`?

Reply:
I've added a comment above these tests to spell out the intended behavior.